test/compile/mark/text.test.ts

```diff
@@ -242,16 +242,10 @@ describe('Mark: Text', () => {
     });
 
     it('should map color to fill', () => {
-      expect(props.fill).toEqual([
-        {
-          test: '!isValid(datum["mean_Acceleration"]) || !isFinite(+datum["mean_Acceleration"])',
-          value: null
-        },
-        {
-          scale: 'color',
-          field: 'mean_Acceleration'
-        }
-      ]);
+      expect(props.fill).toEqual({
```

Comment:
Intended change -- we no longer relying "null" fill to hide invalid marks in the new `"break-paths-keep-domains"` that replaces in the old secret "hide" mode.